### PR TITLE
Update README.md for 1.11

### PR DESCRIPTION
**Pull request by hrydgard**
1.11 is not quite ready but I'm gonna try to finish it up soon: https://github.com/hrydgard/ppsspp/milestone/22

In the meantime, let's write the README update!

In the future, might make sense to have an [Unreleased] section and update it as we go instead...

As usual, I just did a quick skim of the milestone here and surely missed a lot. Please add what you think should be highlighted.

### CODE REVIEW

**Comment by iota97** [comment]
Issues id should be written with no square bracket tho'. The script seek the one that have no [ before to add the link, the title and automatically add the brackets to avoid doing thing more than once. We should just write them "as usual" and run the script once it's all done.

**Comment by sum2012** [comment]
Fixed the following games
Namco Museum - Battle Collection (#9523 )
Namco Museum Vol. 2 - ULJS00047 (#13297)
Dream Club Portable crash after select girl (#6025)
Capcom Classic Collection Reloaded stuck in return game (#4671)
Xyanide Resurrection freezing (#8526)

**Comment by unknownbrackets** [comment]
> Multiple fixes to the IR interpreter

Might mention iOS here, though maybe everyone using it knows?

> and more...

Heh, I think it's already said at the top.  Maybe we just say "and more" for fixed games?

Potentially worth mentioning, although not really significant: #13335 (maybe mention support for custom plugins and improved APIs?), #11795/#13092 (fast memory made more stable?), #13172 (maybe too many to list but this is sorta separate), #12979 (maybe it was already mostly fixed in v1.10.3?, but it's finally seemed stable for me...), #13235 (maybe along with MMPX, I don't think hardware texture scaling was a UI option before?)

-[Unknown]

**Comment by sum2012** [comment]
I forget mention this one.
Downstream Panic (US) New Game crashes (#13633)

**Review by unknownbrackets — APPROVED** [review]

**Comment by zakilj3** [comment]
could also add to the readme that some games multiplayers were fixed like:
dragon ball shin budokai
powerstone
bleach heat the soul 7
kingdom hearts birth by sleep
GTA vice city story

dunno any other fixed multiplayer games

**Comment by sum2012** [comment]
Fixed Hanayaka Nari Wa ga Ichizoku strange MP3 mono voice (#5213)
Princess Maker 5 Portable OpenGL okay in (#13741 ) vulkan still fail
Dream Club portable OpenGL and vulkan okay in (#6497) But D3D9,D3D11 still fail

**Comment by nassau-tk** [comment]
Fix Assassin's Creed : Bloodlines Save issue  on Android (#12761 )

**Comment by Anuskuss** [comment]
This is a big one #7124 (has been open for 6 years before getting fixed).

**Comment by sum2012** [comment]
@hrydgard Don't forget to run script before release v1.11

**Comment by Githubot1957** [comment]
ETA of ppsspp v1.11?

**Comment by Gamemulatorer** [comment]
> ETA of ppsspp v1.11?

Maybe next week? 🤔

**Comment by nassau-tk** [comment]
@hrydgard 

By the way...
Did you fix by some way about priority of kr0.pgf?
https://github.com/hrydgard/ppsspp/issues/13190#issuecomment-758692863

**Comment by hrydgard** [comment]
Support for the real kr0.pgf is not likely to happen anytime soon...

**Review by sum2012 — CHANGES_REQUESTED** [review]

**Comment by nassau-tk** [comment]
> Support for the real kr0.pgf is not likely to happen anytime soon...
> 
> Maybe we should ignore the real one it in favor of ours, if someone has installed it.

@hrydgard 

Yes, sure. I think so too.
So, I want that "our kr0.pgf" must always high priority than "real kr0.pgf".

File: README.md [hunk 1]
```diff
@@ -25,6 +25,45 @@ If you want to download regularly updated builds for Android, Windows x86 and x6
...
+* Desktop: Support for multiple instance multiplayer (#13172, ...)
+* Workaround for rendering bugs with flat shading in iOS 14
+* Multiple fixes to the IR interpreter (#13897, ...)
+* UI: New fullscreen button on desktop platforms, optional navigation sounds (#13239)
+* Audio and multiple hangs fixes in UWP version (#13792, ...)
+* Partial microphone support (#12336, ...)
+* Workaround for wacky action mirroring bug in Hitman Reborn Battle Arena 2 (#13706, #13526)
+* Hardware texture upscaling for Vulkan, mipmap generation (#13235, #13514)
+* Added MMPX Vulkan texture upscaling shader (#13986)
+* Depth texturing support in Vulkan and D3D11 (13667, ...)
```
[the remark this answers is not in the document]
Comment (hrydgard) [reply README.md]: Fixed

**Comment by hrydgard** [comment]
the kr0 problem needs its own issue, doesn't it have one? let's not discuss it here.

**Comment by sum2012** [comment]
chinese translate readme

新的瀏覽...按鈕，允許在Android 11上打開SD卡
* ANR2ME修復了無數AdHoc網絡，例如Dragon Ball Shin Budokai，PowerStone，
 漂白熱魂7，王國之心，GTA：VCS等。
* 修正Outrun, Dirt 2圖形問題
*修正了遊戲王的截止卡（＃7124）。
* nassau-tk對內置字體的大量修復
*添加了異常處理程序，因此如果遊戲崩潰，PPSSPP仍可保持活動狀態（＃11795 /＃13092）
*桌面：支持多實例多人遊戲（＃13172，...）
*在iOS 14中使用平面陰影渲染錯誤的解決方法
*IR解釋器的多個修復程序（＃13897，...）
* UI：桌面平台上的新全屏按鈕，可選的導航聲音（＃13239）
* UWP版本（＃13792，...）中的音頻和多個掛起修復
*部分麥克風支持（＃12336，...）
*解決《殺手重生戰鬥競技場2》（＃13706，＃13526）中古怪的動作鏡像錯誤的解決方法
*用於Vulkan，mipmap生成的硬件紋理升級（＃13235，＃13514）
*添加了MMPX Vulkan紋理放大著色器（＃13986）
* Vulkan和D3D11中的深度紋理化支持（13667，...）
*Test Drive Unlimited（＃13355，...）的性能修復
*允許在手機上快退（＃13866）
*添加了禁用屏幕消息的選項（＃13695）
*在libretro（＃13654）上添加了“效果的較低分辨率”
*允許鏈接多個後處理著色器（＃12924）
*支持加載特定於遊戲的插件（＃13335）
*修復了Android上的刺客信條：血統保存問題（＃12761）
修正遊戲：
 * Namco Museum - Battle Collection, Vol 2 (#9523, # 13297, #13298)
 * Dream Club Portabl1e (目前 GL and Vulkan) (#6025)
 * Capcom Classic Collection OpenReloaded (stuck in return game) (#4671)
 * Xyanide Resurrection (freezing) (#8526)
 * Dissidia Final Fantasy Chinese (patched game, invalid address) (#13204)
 * Crazy Taxi (#13368)
 * Spiderman: Friend or Foe (#13969)
 * Downstream Panic (US) (New Game crash) (#13633)
修正美少女夢工場5 圖形問題,目前只支持OpenGL

**Comment by hrydgard** [comment]
thanks! I'll put it on the website when I finish up the release. having some build machine problems

**Comment by AkiraJkr** [comment]
Portuguese Brazilian translation

Novo na versão 1.11.0
====================
Tem coisas até demais para listar desta vez.
* Várias melhorias e correções de performance, crashes e pequenos bugs.
* Problemas gráficos corrigidos em reflexos de carros em Outrun, Dirt 2 ([#13636], [#13640], [#13760])
* Correções sem fim por "ANR2ME" da rede AdHoc, demais para listar
* Crashes corrigidos em Crazy Taxi ([#13368]), Spiderman: Friend or Foe ([#13969])
* Várias correções das fontes embutidas por "nassau-tk"
* Gambiarra feita para bugs de renderização com shading plano em iOS 14
* Várias correções para o Intérprete IR ([#13897], ...)
* Novo botão Procurar... permite abrir cartões SD no Android 11
* UI: Novo botão fullscreen para plataformas desktop, sons de navegação opcionais
* Correções de suspensões e audio para a versão UWP ([#13792], ...)
* Suporte parcial para microfone ([#12336], ...)
* Gambiarra para um estranho bug de ações sendo "espelhadas" em Hitman Reborn Battle Arena 2 ([#13706], [#13526])
* Adicionado o shader de upscale de textura MMPX para o Vulkan ([#13986])
* Auto-gerar "mipmaps" para a escala de texturas  ([#13514]) (Apenas em Vulkan)
* Suporte para a texturização com profundidade em Vulkan e D3D11 ([13667], ...)
* Correção de performance para Test Drive Unlimited ([#13355], ...)
* Correção das suspensões em Namco Museum ([#13298], ...)
e mais...

**Pull request merged**